task03-one-piece-website/index.html

```diff
@@ -0,0 +1,183 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@400;500;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/style.css">
+  <title>One Piece</title>
+</head>
+<body>
+  <header class="header">
+    <i class="ico-menu header__icon header__icon--menu"></i>
+    <i class="ico-x header__icon header__icon--x"></i>
+    <img class="header__logo" src="../assets/img/op-logo.svg" alt="one piece logo">
+    <div class="header__fill"></div>
+    <nav class="menu">
+      <a href="#" class="menu__link">Home</a>
+      <a href="#" class="menu__link">Characters</a>
+      <a href="#" class="menu__link">Islands</a>
+      <a href="#" class="menu__link">Mist Objects</a>
+      <button class="menu__lang menu__lang--en">
+        <span>English</span>
+        <img src="../assets/img/usa-flag.svg">
+      </button>
+      <button class="menu__lang menu__lang--es">
+        <span>Spanish</span>
+        <img src="../assets/img/spain-flag.svg">
+      </button>
+    </nav>
+  </header>
+
+  <main class="main">
+    <div class="op-characters">
+      <img class="op-characters__img" src="../assets/img/op-characters.svg" srcset="../assets/img/op-characters-big.png 1280w" alt="one piece main characters">
```

Comment:
![image](https://user-images.githubusercontent.com/69167331/158636297-6aac0ad5-3301-4c6e-8c38-c4cc0d6eeedf.png)

Image size if great, however, one of the characters head is not being displayed (left)... It gets cut by the border. Please make sure the image is shown exactly as the figma prototype shows (right).
This happens when the Sinopsis box is expanded and contracted

Reply:
Just noticed this happens on tablet as well